packages/bruno-app/src/utils/curl/parse-curl.spec.js

```diff
@@ -80,6 +80,36 @@ describe('parseCurlCommand', () => {
       });
     });
 
+    it('should parse single header (no space in header value)', () => {
```

Comment:
@TanishqSingla Both the tests are identical.

Reply:
My bad, I didn't notice the duplication

Reply:
Please resolve this conversation if you're okay with the changes